[Ready] Wedding Content! Crate with tuxedo, wedding dress, cummerbund, veil, and craftable bouquets

OnlineGirlfriend opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds a wedding crate which includes wedding dress, tuxedo, cummerbund (on the belt slot), veil, champagne, and three bouquets - mixed, sunflower, and poppy. The cost of the crate is 1500 credits. One each of wedding dress, tuxedo, cummerbund, and veil are available in the autodrobe.

Mixed bouquets = 2 lilies, 2 geraniums, 2 sunflowers
Sunflower bouquets = 6 sunflowers
Poppy bouquets = 6 poppies

I'd like to add a couple more bouquet options in the future.

I also moved Camera Obscura within recipes.dm to be with other misc. items.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Enhances wedding RP! More options for formalwear.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: wedding crate
add: tuxedo, wedding dress, veil, cummerbund, craftable bouquets
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

OnlineGirlfriend committed:
wedding crate

-add craftable bouquet
-slightly reorganize recipes.dm
-add cummerbund
-add wedding crate
-add wedding dress and tuxedo

OnlineGirlfriend committed:
Adds wedding crate and supplies

- Craftable bouquet
- Tuxedo
- Wedding dress
- Cummerbund (belt)
- Icons

SuperNovaa41 commented:
cummerbund

actions-user committed:
Automatic changelog compile [ci skip]

TheFakeElon commented:
cute

CRITAWAKETS commented:
homophobic content, should have two tuxedos and two wedding dresses in the crate in case you wish to perform a gay marriage

scarwolff commented:
fuck this pr, empowering felinids like that, whats wrong with you.

OnlineGirlfriend commented:
> homophobic content, should have two tuxedos and two wedding dresses in the crate in case you wish to perform a gay marriage

that is a good point

OnlineGirlfriend committed:
tweak crate and add to autodrobe

two each of wedding dress and tuxedo in the wedding crate, plus one of each available in the autodrobe

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

tralezab commented:
Why not?

Tlaltecuhtli commented:
coommerbund

ATH1909 commented:
It might be nice to allow any kinds of flowers to count for the bouquet, so that you don't need botany to grow 3 specific plants (or buy a shitton of wedding crates) to get a bunch of bouquets to hand out.

GuyonBroadway commented:
Really this should go under the chapel supplies shouldn't it?

Hell give the chaplain a whole segment to do various ceremonies from weddings to that klingon pain ritual stuff.

GuyonBroadway commented:
Also people are forgetting that weddings are like court cases on ss13.

A perfect antag oppertunity to show off your latest flamethrower mix.

Fikou commented on code/modules/vending/autodrobe.dm:
```diff
@@ -140,7 +140,9 @@
-				   /obj/item/clothing/suit/hooded/mysticrobe = 1)
+				   /obj/item/clothing/suit/hooded/mysticrobe = 1,
+				   /obj/item/clothing/under/dress/wedding_dress = 1,
+				   /obj/item/clothing/under/suit/tuxedo = 1)
```
they shouldnt be above skub...

Tlaltecuhtli replied:
undocumented and shouldnt be here, clowns dont need wedding dress

OnlineGirlfriend replied:
it is in fact documented

Pugie commented:
wedding ring glove slot item would be cool or it could be an attachment to gloves.

ExcessiveUseOfCobblestone commented:
can someone post the screenshot where the guy rationalizes his ERP by saying they're lawfully wedded, thanks.

OnlineGirlfriend committed:
Additional wedding content

-Adds sunflower and poppy bouquets
-Adds wedding veil
-Updates wedding crate contents and cost
-Veil and cummerbund now in autodrobe

OnlineGirlfriend commented:
So thinking about the gay wedding thing, I opted to keep one of each outfit in the crate while lowering the cost so it's more reasonable to order two. The reason for this is that I want additional unique content in the crate, like the bouquets I just added and possibly rings or bridesmaid dresses.

OnlineGirlfriend committed:
tweak veil

OnlineGirlfriend committed:
tweak veil

OnlineGirlfriend committed:
fix icon state

tmtmtl30 commented:
>implying the best part of a gay wedding isn't the visual gag of who wears what

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend commented:
pardon me while I learn how to use a computer

OnlineGirlfriend committed:
Update belt.dmi

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

OnlineGirlfriend committed:
Merge branch 'master' into weddings

OnlineGirlfriend committed:
Update belt.dmi

OnlineGirlfriend committed:
Update tgstation.dme

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

TheFakeElon commented:
Port rings from austation 😳

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

Twaticus reviewed (changes requested):
the tuxedo in icons/obj/clothing/under/suits.dmi is using directionals when it should just be 1 sprite and it needs be cropped to match the other suits in that file.
the cumbund thing in icons/mob/clothing/belt.dmi is missing a name

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend commented:
I will update this tonight or tomorrow, thanks for the feedback!

Also if someone could remove UI label, this doesn't touch UI.

actions-user committed:
Automatic changelog compile [ci skip]

actions-user committed:
Automatic changelog compile [ci skip]

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

OnlineGirlfriend committed:
crop tuxedo obj

OnlineGirlfriend committed:
Update belt.dmi

OnlineGirlfriend committed:
Remove merge conflict

Update head.dmi, hats.dmi, items_and_weapons.dmi

OnlineGirlfriend committed:
Revert "Remove merge conflict"

This reverts commit dc7ea4be37737fbfc974e09f97dee86fceac0776.

OnlineGirlfriend committed:
Update head.dmi

OnlineGirlfriend committed:
update hats.dmi and items_and_weapons.dmi

OnlineGirlfriend committed:
Merge branch 'master' of https://github.com/OnlineGirlfriend/tgstation

OnlineGirlfriend committed:
Merge branch 'master' into weddings

Tumesuo commented:
lol cum
